mdsa/algorithms/fsvd.py

```diff
@@ -0,0 +1,100 @@
+from numpy import dot, hstack
+from numpy.linalg import qr, svd
+from numpy.random import standard_normal
+from mdsa.algorithm import Algorithm
+
+
+class FSVD(Algorithm):
+    def __init__(self):
+        super(FSVD, self).__init__(algorithm_name='fsvd')
+
+    def run(self, distance_matrix, num_dimensions_out=10):
+        """
+        Takes an input matrix and returns eigenvalues and eigenvectors
```

Comment:
Could you edit this docstring to be more similar to [this one](https://github.com/biocore/scikit-bio/blob/b39deca3d60a086017eeea611577ce8648304805/skbio/diversity/beta/_unifrac.py#L28)?